Alarmi/Alarmi/Sources/ViewControllers/TodayViewController.swift

```diff
@@ -10,12 +10,108 @@ import UIKit
 
 class TodayViewController: UIViewController {
 
+    // MARK: View
+    private let stackView: UIStackView = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.axis = .vertical
+        $0.distribution = .fill
+        $0.alignment = .leading
+        $0.spacing = 8
+        return $0
+    }(UIStackView())
+
+    private let descriptionLabel: UILabel = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.text = "마지막으로 전화한지 5일이 지났어요."
+        $0.font = UIFont.preferredFont(forTextStyle: .title2)
```

Comment:
p1;

이렇게만 지정해주면 dynamic types 적용이 안됩니다.
UILabel들에 모두 속성 두 가지 더 붙여주세요.
```suggestion
        $0.font = UIFont.preferredFont(forTextStyle: .title2)
        $0.adjustsFontForContentSizeCategory = true
        $0.numberOfLines = 0
```

dynamic types의 길은 너무 험하네요ㅠㅠ 너무 욕심이었을까요?